### ISSUE TITLE
ETH Miner Doesnt Work

### ISSUE THREAD

**Issue by yukari213** [UnamSanctam/SilentCryptoMiner#308]
Hi, i started mining **ETH** using your program and i think i did something wrong with the settings, because i can see in the checker miner should be active **_but on ethermine there is no activity, same on my web panel_**

![image](https://user-images.githubusercontent.com/49552093/149931045-c1321666-d633-4cf2-9173-d8b2969071b3.png)
![image](https://user-images.githubusercontent.com/49552093/149931096-f5c87498-335a-4fde-92f8-4fb4173cb9a3.png)
![Bez tytułu](https://user-images.githubusercontent.com/49552093/149931320-4c77f782-3fab-4b99-8d49-0fdd7ae87647.png)

I tried to use multiple adresses, diffrent configs from Issues section, etc but nothing makes actual diffrence.
This is my config

![image](https://user-images.githubusercontent.com/49552093/149931557-eee08634-d4a0-4911-91f4-a6c99c911d43.png)
![image](https://user-images.githubusercontent.com/49552093/149931592-7fdcb79b-7ab1-428a-a27d-c88c6b7d5862.png)

![image](https://user-images.githubusercontent.com/49552093/149931639-bb62b593-5d4b-4b58-afca-82f6501838e3.png)

Startup is **disabled**

![image](https://user-images.githubusercontent.com/49552093/149931693-d9857ead-5005-4144-bfbe-dd51b621b4db.png)

Icon same, in options i turned on only Shellcode Loader and DEBUG function _(tried without that before)_

I dont really know what to do i would be gratefull if you help me

**Comment by yukari213** [comment]
I was trying with nanopool again and i saw that miner process shows up for 10-20 seconds and disappears
again in the checker there is still miner active. Web panel still doesnt show up anything

https://youtu.be/nF_DhH3fD4k

**Comment by SergioCupido** [comment]
Hi, you should use programs like msi afterburner to see that eth miner is running. Miners using gpu will not appear in task manager.

**Comment by UnamSanctam** [comment]
Isn't that a 4GB card? In that case you can't mine ETH since it requires at least 4.5GB of VRAM. And is the web panel being hosted on the same computer as the miner or another computer?

**Comment by yukari213** [comment]
Yes, it its 4gb card. Im hosting web panel on vps

**Comment by UnamSanctam** [comment]
Since it's a 4GB card it won't be able to mine (of course) though it should work fine on a 6GB card. If you visit the URL you entered into "API Endpoint URL" do you get a "Error: Invalid Data" (what you're supposed to get)?

**Comment by yukari213** [comment]
yes, i get invalid data text. I started mining eth on 8gb card like 10/15 minutes ago and it still doesnt pop up

**Comment by yukari213** [comment]
i checked now, on nanopool or ethermine says theres no active workers, same on web panel. But graphic card is on 100% usage (8gb)

**Comment by UnamSanctam** [comment]
If the GPU is being used then it's necessarily working (mining to the pool), but on your VPS make sure that the web server folder and unamwebpanel.db both have write permissions so that things can be written to the database.

**Comment by yukari213** [comment]
i gave like all webpanel files 777 permission using chmod in terminal but it still doesnt show up in miner tab

**Comment by UnamSanctam** [comment]
And you changed the owner as well? Can you find any log files inside `__UNAM_LIB\Logs` or any error.log anywhere?

**Comment by yukari213** [comment]
I found dummy file in __UNAM_LIB/Logs 
![image](https://user-images.githubusercontent.com/49552093/150160070-529ba81b-b775-4cc2-ad89-db5a1944549d.png)
and few error logs and acces in var/log/apache2
![image](https://user-images.githubusercontent.com/49552093/150160319-65f4cb92-93bf-42b2-9f34-85f7ac97c4d6.png)
[error.log](https://github.com/UnamSanctam/SilentCryptoMiner/files/7898294/error.log)
error log show that there is no permission for something but im sure i used chmod correctly, files in putty shows in green color so it means it got 777 permission
![image](https://user-images.githubusercontent.com/49552093/150161402-3e3d9dfd-4b8e-4ea7-aaa7-fbff0476f482.png)
![image](https://user-images.githubusercontent.com/49552093/150161473-64fb3a00-3d7a-4a4d-9c0b-17b96cf7dc67.png)

i was trying to mine etc few minutes ago and it shows up on ethermine 
![image](https://user-images.githubusercontent.com/49552093/150161631-ab5b26f1-a56b-4b27-8bbc-ed41726c10c5.png)

**Comment by UnamSanctam** [comment]
Seems like you don't have PDO SQLite installed, the permission errors are because it's not allowed to write into `__UNAM_LIB\Logs`. Try running `sudo apt-get install php-sqlite3` to install the SQLite driver.

**Comment by yukari213** [comment]
nothing new :/
![image](https://user-images.githubusercontent.com/49552093/150162859-64e524ba-db96-4922-add1-880b47a35b9f.png)
![image](https://user-images.githubusercontent.com/49552093/150163063-d90518d3-2540-4779-8acc-b66ed33ace5e.png)

**Comment by UnamSanctam** [comment]
Wait for a bit but otherwise try removing the error logs you can find and then wait a bit and then check if new ones have appeared so that you can see if there are any things not installed and such.

**Comment by yukari213** [comment]
This time it shows up this
[error.log](https://github.com/UnamSanctam/SilentCryptoMiner/files/7898501/error.log)
i dont know which package it is

**Comment by UnamSanctam** [comment]
Can you change the permissions in __UNAM_LIB so that you can get the logs?

**Comment by yukari213** [comment]
Yes, i checked that file and i saw that unamwebpanel.db was read-only 🤣 Ill try to start miner again and close the issue if it will work. Thanks for help <3
